src/vignette/storage/static_assets.clj

```diff
@@ -22,23 +22,27 @@
     (fs/file-exists? to))
   (->response-object [this] (file-stream this)))
 
-(defn create-async-response-stored-object [url]
-  (let [response @(http/get url {:as :stream})]
-    (if (-> response :status (= 200))
-      (->AsyncResponseStoredObject response))))
+(defn create-async-response-stored-object [static-image-get image-review-get]
+  (let [static-image-response @static-image-get]
```

Comment:
Call to image review ensures that image exists and has any of the specified flags, i.e. it checks if the image is safe to show.
So to show the image both services need to agree that image exists and can be shown.